admin/updates/update-3.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "user" (
```

Comment:
`user` is a reserved word in postgres. You get around it by using quotes everywhere, but I can imagine this causing someone lots of pain if they don't realise this (I've been there before)